node/clients/biggy-search.ts

```diff
@@ -161,6 +162,7 @@ export class BiggySearchClient extends ExternalClient {
         operator,
         fuzzy,
         bgy_leap: leap ? true : undefined,
+        ['hide-unavailable-items']: hideUnavailableItems ? 'true' : 'false',
```

Comment:
hideUnavailableItems can be undefined? In case it can't, this line could be like: `['hide-unavailable-items']: hideUnavailableItems`